src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsHashCodeCheck.java

```diff
@@ -49,7 +49,60 @@
  * <pre>
  * &lt;module name=&quot;EqualsHashCode&quot;/&gt;
  * </pre>
- *
+ * <p>Example:</p>
+ * <pre>
+ * public static class Example1 {
+ *     public int hashCode() {
+ *         //code
+ *     }
+ *     public boolean equals(String o) { // violation, no correct `equals`
+ *         //code
+ *     }
+ * }
+ * public static class Example2 {
+ *     public boolean equals(Object o) { // violation, no `hashCode`
+ *         //code
+ *     }
+ *     public boolean equals(String o) {
+ *         //code
+ *     }
+ * }
+ * public static class Example3 { // ok
```

Comment:
Please move `// ok` comments to methods, to make it similar to violation comments